packages/phone-number-privacy/signer/src/pnp/endpoints/sign/action.ts

```diff
@@ -39,7 +39,8 @@ export class PnpSignAction implements Action<SignMessageRequest> {
         // may be undefined.
         // In the case of a database connection failure, queryCount
         // may be undefined.
-        if (quotaStatus.queryCount && quotaStatus.totalQuota) {
+        // Note that queryCount or totalQuota can be 0 and that should not fail open.
+        if (quotaStatus.queryCount !== undefined && quotaStatus.totalQuota !== undefined) {
```

Comment:
@alecps lmk if this looks ok to you; would be great to capture this change in signer PNP_SIGN tests as well if possible! 🙏

Reply:
I added a TODO to completely revisit this logic entirely haha

Reply:
This looks okay for now

Reply:
sg!